react/LocationSearch.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { Input, IconSearch, IconClear } from 'vtex.styleguide'
+import { FormattedMessage } from 'react-intl'
+
+import styles from './LocationSearch.css'
+
+const LocationSearch: React.FC = () => {
+  const [inputValue, setInputValue] = useState<string>('')
+
+  const onClear = () => setInputValue('')
+
+  return (
+    <div className={`${styles.locationSearch} w-100`}>
+      <Input
+        label={
+          <FormattedMessage id="place-components.label.autocompleteAddress" />
+        }
+        prefix={<IconSearch color="#134CD8" />}
+        suffix={
+          inputValue.length && (
+            <span
+              role="button"
+              tabIndex={-1}
+              className="pointer"
+              onClick={onClear}
+              onKeyPress={onClear}
```

Comment:
you need to check for the key in this event (either `enter` or `space`) to trigger the `onClear` function, else we could tab into this element and hit `tab` to go to the next tabbable element and end up clearing the input, which wouldn't be intended